src/control/drpc/modules.go

```diff
@@ -134,7 +134,7 @@ func (m MgmtMethod) String() string {
 		MethodPoolExclude:          "PoolExclude",
 		MethodPoolDrain:            "PoolDrain",
 		MethodPoolExtend:           "PoolExtend",
-		MethodPoolReintegrate:      "PoolReintegrate",
+		MethodPoolReint:            "PoolReint",
```

Comment:
Why the rename? The other methods generally use full words, and `Reint` is less clear than `Reintegrate` without context. Plus, it makes this PR much noisier than it would be without the rename.

Reply:
FWIW, I think partly why long names stand out in C code is because of the conventions used in DAOS. E.g. the DAOS C code tends to do
```
my_return_var = some_long_function_name(some_long_argument_name1,
					some_long_argument_name2,
					some_long_argument_name3)
```
Whereas, IMO, for this reason and several others actually, I think this is much more reasonable
```
my_return_var = some_long_function_name(
	some_long_argument_name1,
	some_long_argument_name2,
	some_long_argument_name3)
```

Reply:
We all come from different programming experiences and it's always about striking a balance but I generally find that people coming from C backgrounds are uncomfortable with long variable names which people from other backgrounds may be more comfortable with. So when interacting with C engine code I tend to try to be considerate to the preferences of the relevant code owners. "Reint" is an abbreviation that's been used variously around
the code base and is relatively intuitive and unambiguous. @mjmac are you requesting that I revert the reame?

Reply:
Fair point with regard to the C code, but why does the Go code need to be truncated, too? Go norms focus on readability and maintainability. I don't have a problem with the C code conforming to the style in that codebase, but I do think the Go code should be reverted to the more verbose and readable conventions.

Reply:
ok